recipes/perl-class-accessor/recipe.yaml

```diff
@@ -0,0 +1,50 @@
+context:
+  version: 0.51
+
+package:
+  name: perl-class-accessor
+  version: ${{ version }}
+
+source:
+  url: https://cpan.metacpan.org/authors/id/K/KA/KASEI/Class-Accessor-${{ version }}.tar.gz
+  sha256: bf12a3e5de5a2c6e8a447b364f4f5a050bf74624c56e315022ae7992ff2f411c
+
+build:
+  number: 0
+  noarch: generic
+  script: |
+    perl Makefile.PL INSTALLDIRS=site NO_PERLLOCAL=1 NO_PACKLIST=1
+    make
+    make test
+    make install VERBINST=1
+    pod2text ${PREFIX}/lib/perl5/core_perl/pod/perlartistic.pod > ${RECIPE_DIR}/perlartistic.txt
+    pod2text ${PREFIX}/lib/perl5/core_perl/pod/perlgpl.pod > ${RECIPE_DIR}/perlgpl.txt
+
+requirements:
+  build:
+    - make
+  host:
+    - perl  # requires perl feedstock update for all versions
+  run:
+    - perl
```

Comment:
`noarch: generic` don't work the same with unix and windows AFAIR. You'd need https://github.com/conda-forge/perl-feedstock/pull/68

Reply:
Are we blocked until that is accepted?